spec/models/asset_spec.rb

```diff
@@ -216,5 +216,10 @@
       asset.destroy
       expect(asset.deleted_at).not_to be_nil
     end
+
+    it "can be restored" do
+      asset.restore
```

Comment:
`asset.deleted_at` is already `nil` here so this test will pass if `asset.restore` does nothing.